src/http/Response.ts

```diff
@@ -1,23 +1,12 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License.
 
-import { Cookie } from '@azure/functions';
-import { HeaderName, MediaType } from '../constants';
+/* eslint-disable deprecation/deprecation */

```

Comment:
I'm okay disabling this rule on a per-file basis for a _test_ file, but not a _src_ file. Please disable on a per-line basis in src